sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -105,6 +111,14 @@
           <statelessTestsetReporter implementation="org.apache.maven.plugin.surefire.extensions.junit5.JUnit5Xml30StatelessReporter">
             <usePhrasedTestCaseMethodName>true</usePhrasedTestCaseMethodName>
           </statelessTestsetReporter>
+          <properties>
+            <configurationParameters>
+              junit.jupiter.conditions.deactivate = *
+              junit.jupiter.extensions.autodetection.enabled = true
+              junit.jupiter.testinstance.lifecycle.default = per_class
+              junit.jupiter.execution.parallel.enabled = true
+            </configurationParameters>
+          </properties>
```

Comment:
Let's aim to remove these on-off configurations from children POMs where possible.

Reply:
Yes, ideally, this surefire plugin should not be in these modules. It should be configured only in parent.

Reply:
This was removed.